tests/TestCase/Http/Cookie/CookieCollectionTest.php

```diff
@@ -436,7 +436,7 @@ public function testCreateFromHeader()
         $header = [
             'http=name; HttpOnly; Secure;',
             'expires=expiring; Expires=Wed, 15-Jun-2022 10:22:22; Path=/api; HttpOnly; Secure;',
-            'expired=expired; Expires=Wed, 15-Jun-2015 10:22:22;',
+            'expired=expired; version=1; Expires=Wed, 15-Jun-2015 10:22:22;',
```

Comment:
Should the test ensure that this attribute isn't in the Cookie object that is created?

Reply:
Sorry!!
I think so.